lib/reports.dart

```diff
@@ -19,17 +19,10 @@ class _ReportsState extends State<ReportsPage> {
             child: SingleChildScrollView(
       child: Column(
         children: <Widget>[
-          SizedBox(
-            height: 600,
-            child: UserReportTable(),
-          ),
-          const Divider(
-            thickness: 1.0,
-          ),
-          SizedBox(
-            height: 600,
-            child: ReportsGraph(),
-          )
+            SizedBox(
+              height: 600,
```

Comment:
Just found out that you can do `MediaQuery.of(context).size.height` instead of defining a set value of 600 here. This will fill entire available space and look better.